File: src/main/java/no/nav/pensjon/regler/domain/beregning2011/Garantitillegg.java
package no.nav.pensjon.regler.domain.beregning2011;

import no.nav.pensjon.regler.domain.beregning.Ytelseskomponent;
import no.nav.pensjon.regler.domain.kode.YtelsekomponentTypeCti;

public class Garantitillegg extends Ytelseskomponent {
    private static final long serialVersionUID = -222566587860410921L;

    public Garantitillegg() {
        super();
        ytelsekomponentType = new YtelsekomponentTypeCti("GAT");
    }

    public Garantitillegg(Garantitillegg gt) {
        super(gt);
        ytelsekomponentType = new YtelsekomponentTypeCti("GAT");
    }

}

File: src/main/kotlin/no/nav/pensjon/regler/domain/beregning2011/Garantitillegg.kt
package no.nav.pensjon.regler.domain.beregning2011

import no.nav.pensjon.regler.domain.beregning.Ytelseskomponent
import no.nav.pensjon.regler.domain.enum.YtelseskomponentTypeEnum

class Garantitillegg : Ytelseskomponent() {
    override val ytelsekomponentTypeEnum: YtelseskomponentTypeEnum = YtelseskomponentTypeEnum.GAT
}